src-ts/tools/learn/welcome/course-title/CourseTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react'
+
+import { LearnChallengeBadgeIcon } from '../../../../lib'
```

Comment:
i know i haven't been very good about this, but we should probably only put icons in the lib if they are reusable. 

This icon is clearly specific to the learn tool, so we either need to rename it to be more generic or move it to the learn tool directory.